test/navigation/nonsipp/LandOrPropertyNavigatorSpec.scala

```diff
@@ -115,6 +115,16 @@ class LandOrPropertyNavigatorSpec extends BaseSpec with NavigatorBehaviours {
         .withName("go from land or property when did scheme acquire page to land property independent valuation page")
     )
 
+    act.like(
+      normalmode
+        .navigateToWithIndex(
+          index,
+          LandPropertyIndividualSellersNamePage,
+          (srn, index: Max5000, _) => controllers.routes.UnauthorisedController.onPageLoad()
```

Comment:
Hi, it should go to this page  'controllers.nonsipp.landorproperty.routes.IndividualSellerNiController.onPageLoad(srn, index, NormalMode)'  instead of 'Unauthorised'.